xbmc/settings/lib/SettingSection.h

```diff
@@ -69,6 +121,10 @@ class CSettingGroup : public ISetting
   void AddSettings(const SettingList &settings);
 
 private:
+  int m_label;
```

Comment:
Maybe it's time to move `m_label`, `m_help` and all their getters and setters into `ISetting` since it's now used by all of its subclasses (`CSettingSection`, `CSettingCategory`, `CSettingGroup` and `CSetting`).